src/Symfony/Component/Config/Tests/Builder/GeneratedConfigTest.php

```diff
@@ -48,12 +49,16 @@ public function fixtureNames()
      */
     public function testConfig(string $name, string $alias)
     {
+        $outputDir = sys_get_temp_dir().\DIRECTORY_SEPARATOR.uniqid('sf_config_builder', true);
```

Comment:
What about deleting this directory on `tearDown`?

Reply:
done